.mkosi/mkosi.fedora

```diff
@@ -72,6 +72,8 @@ Packages=
         e2fsprogs
         libidn2
         libseccomp
+        # libzstd can be dropped once the Fedora RPM gets a dependency on it
+        libzstd
         procps-ng
         util-linux
```

Comment:
systemd.rpm requires
coreutils
libcryptsetup.so.12()(64bit)
libkmod.so.2()(64bit)
libidn2.so.0()(64bit)
libseccomp.so.2()(64bit)
util-linux
so if that comment is true, most of the items here can be dropped.

For the other ones:
e2fsprogs
procps-ng
a comment would be nice why they are required.

Reply:
So it appears that things work (or at least the steps in HACKING.md work) with only libzstd in the package list.

https://github.com/systemd/mkosi/commit/b24bc4a51b68148e516c30c191682a171c1f387d is included in the v5 release of mkosi so if that's the minimum supported version I guess e2fsprogs could go away assuming it's only necessary for bootable images.

procps-ng does get pulled in even when not listed in packages (I've not investigated the dependency tree) but it does seem like it should be explicitly declared.

The rest of them do appear to get pulled in, but they were added rather recently in 64db6f3644c3f889eb4f961f5d7dfebce1fc37ef which makes it sound like they didn't used to.

All that to say I agree with you, I just don't know enough to say if dropping them breaks some build configuration I don't know about yet.

Reply:
hmm, so i was sure i built a fedora image when i added those lines that where too minimal to just work. that's why I added them. If they aren't necessary after all, by all means remove them again. This is developer focussed: let's just remove it, and if someone notices things we can readd them, no need to be extra careful here. hence, @jeremyline can you just drop any package that doesn't appear necessary in your patch?

Reply:
hmm, but keep procps-ng in, it's just too confusing if "ps" isn't available when debugging stuff. And we mark the build bootable, hence e2fsprogs should stay too i guess.

Reply:
Okay, I've left procps-ng with a comment. Since mkosi has logic to add e2fsprogs to images marked bootable using ext4 I dropped that from the package list here.